src/components/WorkspaceAdd.js

```diff
@@ -87,7 +87,7 @@ export class WorkspaceAdd extends React.Component {
           >
             <AppBar position="absolute" color="secondary" onClick={() => (this.setAddResourcesVisibility(false))}>
               <Toolbar>
-                <IconButton className={classes.menuButton} color="secondary" aria-label={t('closeMenu')}>
```

Comment:
@cbeer should this label be replaced instead of being removed ? I think #2050 restores this button, which closes the Add Resources panel. 

![Close add resources panel](https://user-images.githubusercontent.com/5402927/53838085-180a4700-3f49-11e9-8be2-78b0dbf048f7.png)


Does this also need a more specific label, e.g. `Close resources panel` @jvine?

Reply:
👍 ; I've also go ahead and made the label more specific.